dashboard/app/models/levels/blockly.rb

```diff
@@ -398,4 +389,11 @@ def fix_examples
     return if examples.nil?
     self.examples = examples.select(&:present?)
   end
+
+  # This should just return false, but we have a few levels.js levels that use
+  # droplet (starwars, and a few test levels). They all have level records of
+  # type 'Blockly', so they can't override this as needed
+  def uses_droplet?
+    %w(MazeEC ArtistEC Applab StudioEC Gamelab).include? name
```

Comment:
Should this be `type` instead of `name`?

Reply:
Ah whoop, it should be `game.name`.